### PR TITLE
ISAICP-6563: Disable bullet/numbered lists and block quote from the Abstract's toolbar.

### PR DESCRIPTION
**Pull request by saidatom**

### CODE REVIEW

**Review by pfrenssen — CHANGES_REQUESTED** [review]
This is looking good but it lacks tests. I will add them, I saw that our coverage of the editor can use some attention.

File: config/sync/user.role.authenticated.yml [hunk 1]
```diff
@@ -102,6 +102,7 @@ permissions:
   - 'update paragraph content accordion_item'
   - 'update paragraph content content_listing'
   - 'update paragraph content simple_paragraph'
+  - 'use text format abstract_html'
```
[the remark this answers is not in the document]
Comment (pfrenssen) [reply config/sync/user.role.authenticated.yml:105]: @saidatom do you agree to rename this to `simple_html`? Or some other suggestion?

[the remark this answers is not in the document]
Comment (saidatom) [reply config/sync/user.role.authenticated.yml:105]: @pfrenssen yes `simple_html` sounds good.

File: config/sync/filter.format.abstract_html.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      responsive: true
+  media_embed:
+    id: media_embed
+    provider: media
+    status: false
+    weight: 100
+    settings:
+      default_view_mode: default
+      allowed_media_types: {  }
+      allowed_view_modes: {  }
```
[the remark this answers is not in the document]
Comment (saidatom) [reply config/sync/filter.format.abstract_html.yml:61]: With `status: false` is not showing.

File: config/sync/user.role.authenticated.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (pfrenssen) [reply config/sync/user.role.authenticated.yml:105]: Argh `simple_html` is already taken! Going to go for `essential_html`.

**Comment by saidatom** [comment]
Looks good. Approved.

**Pull request merged**